packages/commercetools/theme/pages/Category.vue

```diff
@@ -236,17 +236,23 @@ import {
 
 export default {
   transition: 'fade',
-  setup () {
-    const { categories, search, loading } = useCategory()
+  setup (props, { ssrContext }) {
+    const { search, categories, loading, ssrData } = useCategory(ssrContext)
+    const products = reactive([])
 
-    search({ slug: "men" })
+    onServerPrefetch(async () => {
+      ssrData.products = products // I can create for this helper function like: attachData(ssrData, { products }) if you like the idea
```

Comment:
Let's talk about this? This is required as we pass reference to ssrData object.
Any changes on ssrData will affect on SSR side and will be passed to the client